src/app/fdctl/run/tiles/metrics.c

```diff
@@ -0,0 +1,392 @@
+#include "../../fdctl.h"
+#include "../run.h"
+
+#include "../../../../disco/fd_disco.h"
+#include "../../../../util/metrics/fd_metrics.h"
+#include "../../../../util/metrics/fd_metrics_impl.h"
+#include <stdio.h>
+#include <errno.h>
+
+#include <linux/unistd.h>
+#include <arpa/inet.h>
+#include <openssl/ssl.h>
+#include <openssl/err.h>
+#define __USE_MISC
+#include <netdb.h>
+
+#define DEBUG 0
+
+#define METRICS_TAG 0x8bf72a57ca2947b7UL
+
+static void
+ssl_init( void ) {
+  SSL_library_init();
```

Comment:
Are these all necessary during privileged boot? Can you add some comments. Also error checking, same as https://github.com/firedancer-io/firedancer/blob/b0f14a28f9a4f387ab35c481760d31a8c24cf3e1/src/app/fdctl/run/tiles/quic.c#L33

Reply:
They are - none of them have a return value.